test/test_srtp.c

```diff
@@ -0,0 +1,86 @@
+/*
+ * Tests specific.
+ */
+#include "cutest.h"
+
+/*
+ * libSRTP specific.
+ */
+#include "../srtp/srtp.c" // Get access to static functions
+
+/*
+ * Standard library
+ */
+
+/*
+ *Forward declaration of all tests
+ */
+
+void srtp_calc_aead_iv_srtcp_zero_happy_case();
+// void srtp_calc_aead_iv_srtcp_seq_num_over_0x7FFFFFFF_bad_param();
+
+/*
+ * NULL terminated array of tests.
+ */
+
+TEST_LIST = {{"srtp_calc_aead_iv_srtcp_zero_happy_case()",
+              srtp_calc_aead_iv_srtcp_zero_happy_case},
+             /* {"srtp_calc_aead_iv_srtcp_seq_num_over_0x7FFFFFFF_bad_param()",
+              srtp_calc_aead_iv_srtcp_seq_num_over_0x7FFFFFFF_bad_param}, */
+             {NULL} /* End of tests */};
+
+/*
+ * Implementation
+ */
+
+void srtp_calc_aead_iv_srtcp_zero_happy_case()
```

Comment:
I had to look at the code to determine what this did, not sure I can come up with a better name though, maybe just say what it does ie,   srtcp_calc_aead_iv_srtcp_index_zero ....